torchvision/prototype/models/vision_transformer.py

```diff
@@ -43,6 +43,7 @@ class ViT_B_16_Weights(WeightsEnum):
             **_COMMON_META,
             "num_params": 86567656,
             "size": (224, 224),
+            "min_size": (224, 224),
```

Comment:
Note: Looks good, leave this as-is. We might add more transformer weights on the near future with different sizes and this is why the `size` is not moved on `_COMMON_META`.